include/superior_mysqlpp/dnsa_connection_pool.hpp

```diff
@@ -201,21 +239,34 @@ namespace SuperiorMySqlpp
                 jobThread = std::thread{&DnsAwarePoolManagement::job, std::cref(*this)};
             }
 
+            /**
+             * Returns true if jobThread is in joinable state.
```

Comment:
IMHO point of this function is checking if management thread is running - how it does so is implementation detail. See comment below on discussion about implementation.